.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,7 +7,15 @@ Describe your proposed changes here.
 - [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
 - [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
 - [ ] Rebased/mergeable
+- [ ] Root cause description
+   * Describe the root cause of the bug. 
+   * Did the fix introduce the change in behavior.
+   * What was done to fix the problem. 
 - [ ] Tests pass
+   - [ ] Unit tests
+   - [ ] Integration tests. 
+      * The buf fix affects the interface between components
+   - [ ] System tests. (The bug fix requires testing of the entire system in an environment as close as possible to the real production.)
```

Comment:
Can you: 
- remove the period after `tests`
- update the format of the description for `System tests` so it looks similar to the prior description for `Integration tests`? Below `System tests` add a bullet point then the description `The bug fix requires testing...`

Reply:
done